packages/graphback-codegen-schema/src/SchemaCRUDPlugin.ts

```diff
@@ -281,6 +283,43 @@ export class SchemaCRUDPlugin extends GraphbackPlugin {
     schemaComposer.Query.addFields(queryFields)
   }
 
+  protected addMetaFields(schemaComposer: SchemaComposer<any>, models: ModelDefinition[]) {
+    models.forEach((model: ModelDefinition, index: number)=> {
+      const name = model.graphqlType.name; 
+      const modelTC = schemaComposer.getOTC(name);
+      const desc = model.graphqlType.description;
+      if (parseMarker("versioned", desc) || parseMarker("delta", desc)) {
```

Comment:
should `versioned`, `delta`, `createdAt` etc be extracted somewhere e.g in a `data-sync.ts` file?

Reply:
I am not sure if I understand, could you elaborate :)

Reply:
WDYT @ssd71 ?

Reply:
By extracting them do you mean parsing them and processing them separately? Just making sure we are on same page, do you mean that we should have single function `parseDataSync` that would be called in `addMetadataFields` ?

Reply:
Sorry, bad wording. I simply meant instead of having hard coded constants, we could refactor the code a bit by extracting the constants into meaningful variable e.g `const createdAt = "createdAt"` etc and use it throughtout the codebase, same for the other constants. Just a nitpick, nothing major. If that sounds okay with you?

Reply:
Cool! Sounds great!

Reply:
How does this look:
https://github.com/aerogear/graphback/blob/da8e46fc7b61fe76f974df4b2eec9f0ef5441e94/packages/graphback-codegen-schema/src/metadataAnnotations.ts#L1-L11

Reply:
delta will be not in the core. We will rename `offix` plugin to `datasync` and use delta there exclusively.

Reply:
Cool, so we remove it from here as well?
https://github.com/aerogear/graphback/blob/da8e46fc7b61fe76f974df4b2eec9f0ef5441e94/packages/graphback-codegen-schema/src/SchemaCRUDPlugin.ts#L292

Reply:
https://github.com/aerogear/graphback/blob/3eb2e6591edf0738a470702dbb22aea116e13bef/packages/graphback-codegen-schema/src/SchemaCRUDPlugin.ts#L292-L293
Now this only works for `@versioned`
WDYT @wtrocki